src/types.ts

```diff
@@ -108,6 +108,9 @@ export interface TramOneElement extends Element {
 	[TRAM_TAG_STORE_KEYS]: string[];
 }
 
+export type TramOneHTMLElement = TramOneElement & HTMLElement;
+export type TramOneSVGElement = TramOneElement & SVGElement;
+
```

Comment:
new type definitions, because TramOneElement is too generic, and in reality (right now) it's always an HTMLElement or SVGElement.

If we ever expose registerDOM, people should be able to make their own TramOneElement definitions (which shouldn't be too hard to do).